src/cmakeTools.ts

```diff
@@ -636,19 +636,21 @@ export class CMakeTools implements api.CMakeToolsAPI {
                             };
                             const cmakeListsFile = await vscode.window.showOpenDialog(openOpts);
                             if (cmakeListsFile) {
+                                // Keep the absolute path for CMakeLists.txt files that are located outside of the workspace folder.
                                 selectedFile = cmakeListsFile[0].fsPath;
                             }
                         } else {
-                            selectedFile = selection.fullPath;
+                            // Keep the relative path for CMakeLists.txt files that are located inside of the workspace folder.
```

Comment:
Can you update this comment to remind that the relative path is stored in the label?